src/TributeSearch.js

```diff
@@ -118,6 +119,16 @@ class TributeSearch {
         return rendered
     }
 
+    continueTraversal(patternCache, index) {
+      if (!this.tribute.exactMatch || patternCache.length < 1) {
+        return true;
+      } else if (this.tribute.exactMatch && patternCache[patternCache.length -1] + 1 === index) {
+        return true;
+      } else {
+        return false;
+      }
+    }
```

Comment:
The `else` statements are unnecessary as each branch returns. So, you could simplify this:

```javascript
      if (!this.tribute.exactMatch || patternCache.length < 1) {
        return true;
      }
      if (this.tribute.exactMatch && patternCache[patternCache.length -1] + 1 === index) {
        return true;
      }
      return false;
```
So, this is nitpicking, and you can choose to ignore it if you want :)

Reply:
No that is fair! And easy, updated :+1: